stdlib/3/sys.pyi

```diff
@@ -122,7 +122,7 @@ def getcheckinterval() -> int: ...  # deprecated
 def getdefaultencoding() -> str: ...
 def getdlopenflags() -> int: ...  # Unix only
 def getfilesystemencoding() -> str: ...  # cannot return None
-def getrefcount(object) -> int: ...
+def getrefcount(object: Any) -> int: ...
```

Comment:
Could you rename the argument? E.g. to `arg`.

Also please do the 2.7 version too.